scripts/pylintrc

```diff
@@ -59,6 +59,8 @@ score=no
 disable=all
 # Identifiers are in the same order as in 'pylint3 --list-msgs'
 enable=
+    fatal,
+#    E, # maybe some day
```

Comment:
Think this comment might be confusing to people. Maybe it could be removed. Could put a longer comment somewhere else that explains what E is, why it can't be enabled, etc., instead.

Reply:
ok I'll remove it.

Reply:
I think removing it is less preferable to keeping it with an explanation of what it would do if uncommented in this case. I'm normally dead set against commented-out code, but your point that it's confusing syntax at the bottom of a manpage is well taken.